apps/client/src/__mocks__/context.js

```diff
@@ -11,7 +11,7 @@ export default ({
   const context = useContext(Context);
   context.topic = topics.find((t) => t.slug === topicMock);
   if (addressMock) {
-    context.address = addressMock;
+    context.autofillData.address = addressMock;
```

Comment:
IMHO I prefer `address` over `autofillData` because address is way more clear what it is and autofillData requires background info (for new users and reviewers). Can we not rename to `userData` or something more general?

Reply:
why not put it straight in the context? 
and just get const { autofillAddress } = useContext(Context); ?
 
And i thought we are getting the address for each activity?

Reply:
Discussed this in today's dev-meeting. We'll refactor the context-variables later.